deployment/upgradeLive.sh

```diff
@@ -21,6 +20,7 @@ if [ ! -d "$1" ] ; then
         unlink /opt/kaltura/liveController/latest
         ln -s /opt/kaltura/liveController/$1 /opt/kaltura/liveController/latest
         ln -s /opt/kaltura/liveController/latest/serviceWrappers/linux/kLiveController /etc/init.d/kLiveController
+        bash /opt/kaltura/liveController/$1/deployment/get_bins "$1"
```

Comment:
You have to make sure all the new scripts you create and push have exec premission -> chmod +x
Guy and I saw that the files downloaded from github do not have this permission which will prevent them from running in the script.